lib/schemas/operationContext.js

```diff
@@ -1,9 +1,11 @@
 /*!
- * Copyright (c) 2019-2020 Digital Bazaar, Inc. All rights reserved.
+ * Copyright (c) 2019-2021 Digital Bazaar, Inc. All rights reserved.
  */
 'use strict';
 
 module.exports = {
   type: 'string',
-  enum: ['https://w3id.org/security/v2']
+  enum: [
+    'https://w3id.org/security/v2'
```

Comment:
Can this be pulled in from `security-context` as well?

Reply:
addressed here c3e1c23